Week3/script.js

```diff
@@ -72,24 +72,22 @@ function threeArguments(a, b, c) {
 
         // 2
 
-function colorCar(car) {
-  return car;
+function colorCar() {
```

Comment:
Ah, now the function `always` returns `a red car`. Can you change the code so that the color that it will print out can be passed in the function. So something like this:

```
function  colorCar(color) {
 //do something with the color to print "a _ car"
}
```
```